src/components/ButtonLink/ButtonLink.tsx

```diff
@@ -36,11 +36,18 @@ export const ButtonLink = ({
       }}
       cursor={disabled ? "not-allowed" : "pointer"}
       style={{
+        display: "inline",
```

Comment:
question: This is a change for a global component, will this work with all the other places we have used it?

Reply:
Link is semantically an `inline` component.